### PR TITLE
Alejandro Macari - Assignment 4

### PR DESCRIPTION
**Pull request by macari-dev**

### CODE REVIEW

**Comment by macari-dev** [comment]
Assignment 2 is actually Assignment 3 sorry for the typo. The delay on assignment 3 is because I only uploaded it on moodle

File: Assignment4/macari-dev-YA6921023/task06.py [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+Automatically generated by Colaboratory.
+
+Original file is located at
+    https://colab.research.google.com/drive/12nUaD1TBMkUc8F2n4_EHLwLJnobVPvBJ
+
+**Task 06: Modifying RDF(s)**
+"""
+
+!pip install rdflib
```
Comment (dgarijo) [inline Assignment4/macari-dev-YA6921023/task06.py:12]: This won't work in Python. It's notebook magic

File: Assignment4/macari-dev-YA6921023/task07.py [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+ns = Namespace("http://somewhere#")
+for s, p, o in g.triples((None, RDFS.subClassOf, ns.Person)):
+  print(s)
+
+###
+
+query = prepareQuery("""
+  SELECT ?subject
+  WHERE {
+    ?subject rdfs:subClassOf/rdfs:subClassOf*  <http://somewhere#Person>.
```
Comment (dgarijo) [inline Assignment4/macari-dev-YA6921023/task07.py:39]: equivalent to subclassOf+

File: Assignment4/macari-dev-YA6921023/task07.py [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+    for ss, pp, oo in g.triples((None, RDF.type, s)):
+      print(ss)
+
+for s, p, o in g.triples((None, RDF.type, ns.Person)):
+  print(s)
+
+query = prepareQuery("""
+  SELECT ?subject
+  WHERE {
+    ?subject rdf:type/rdfs:subClassOf <http://somewhere#Person>.
```
Comment (dgarijo) [inline Assignment4/macari-dev-YA6921023/task07.py:62]: incomplete. You need subclassOf* to retrieve the instances of any subclass of person (or its subclasses)

**Review by dgarijo — CHANGES_REQUESTED** [review]

**Comment by macari-dev** [comment]
Assinment 4 revisions pushed.

**Review by dgarijo — APPROVED** [review]

**Pull request merged**